docs/release-notes/deprecate-modularity.rst

```diff
@@ -0,0 +1,12 @@
+:Type: Kickstart
+:Summary: Deprecate RPM modularity module
+
+:Description:
+    Based on the DNF team recommendation we decided to deprecate RPM modularity
+    feature in Anaconda. This is only a deprecation not removal and we would
+    evaluate RHEL-11 status when it's near based on decisions from the DNF team.
```

Comment:
These are release notes for Fedora and not used for RHEL as far as i know. I think it's correct to remove the RHEL mention, and add a Fedora equivalent if applicable.

Reply:
Amended